operators/jump_to_cut.py

```diff
@@ -16,8 +16,15 @@ class JumpToCut(bpy.types.Operator):
         'name': doc_name(__qualname__),
         'demo': '',
         'description': doc_description(__doc__),
-        'shortcuts': [],
-        'keymap': 'Frames'
+        'shortcuts': [
+            ({'type': 'UP', 'value': 'PRESS'},
+             {'direction': 'forward'},
+             'Jump to next cut or keyframe'),
+            ({'type': 'DOWN', 'value': 'PRESS'},
+             {'direction': 'forward'},
```

Comment:
Both shortcuts go forward

Reply:
Fixed in 5b76092530d1b24a34f9d3b45de5f6cf384fc4fb 😄